index.js

```diff
@@ -138,13 +138,26 @@ var Node = function (path, opts, disposable) {
     daemonPid: function () {
       return this.subprocess && this.subprocess.pid
     },
-    getConf: function (key, cb) {
+    getConfig: function (key, cb) {
+      if (typeof key === 'function') {
+        cb = key
+        key = ''
+      }
       var t = this
       var result = ''
       run(IPFS_EXEC, ['config', key], {env: t.env})
         .on('error', cb)
         .on('data', function (data) { result += data })
-        .on('end', function () { cb(null, result.trim()) })
+        .on('end', function () {
+          cb(null, result.trim())
```

Comment:
that would be nice, but also break current expectations. having both as a option would be ideal